core/templates/dev/head/collection_editor/collection_editor.html

```diff
@@ -25,51 +25,74 @@
   {{dependencies_html}}
 {% endblock header_js %}
 
+{% block navbar_breadcrumb %}
+  <div ng-controller="CollectionEditor">
```

Comment:
Yeah, this isn't good since it will duplicate the state of the ``CollectionEditor`` controller. Do we need separate blocks for the navbar and top nav options (the block immediately below this)? If not, it would probably be preferable to put the directives underneath the single ``CollectionEditor`` controller below, inside the main content block. Since we're writing a new editor, we don't need to frame this in the same way as the exploration editor. Besides, we don't really use these blocks in a major way; our code reuse is done using Angular in the frontend, rather than Jinja in the backend.